test/README.md

```diff
@@ -12,14 +12,9 @@ Run `make build` and migrate the db (this only needs to be done once) to the lat
 
 ```
 PGHOST=localhost PGPORT=5432 PGUSER=postgres PGPASSWORD=foobar PGDATABASE=imagebuilder
-MIGRATIONS_DIR=/home/sanne/workstuff/image-builder/internal/db/migrations/ ./image-builder-migrate-db
+MIGRATIONS_DIR=/$HOME/image-builder/internal/db/migrations/ ./image-builder-migrate-db
```

Comment:
It seems `$HOME` already contains the first slash, so I'd remove the one after the equal sign.